packages/ui/Touchable/Hold.js

```diff
@@ -49,6 +49,17 @@ class Hold {
 		}
 	}
 
+	// This method will get the `onHold`, `onHoldEnd`, `onHoldPulse` props and update the existing `holdConfig` values. So basically update the gesture (hold, flick, drag) props during a gesture callback.
+	updateProps = ({onHold, onHoldEnd, onHoldPulse}) => {
+		// `holdConfig` might not exists a gesture not in progress. Check if gesture exists before updating the references to the `holdConfig`
+		if (!this.holdConfig) return;
```

Comment:
Let's use `!this.isHolding()` for consistency and so we can update that method with new conditions as needed.